recipes/protobuf/all/patches/upstream-pr-7288-android-log.patch

```diff
@@ -0,0 +1,25 @@
+--- a/cmake/libprotobuf.cmake
```

Comment:
```suggestion
https://github.com/protocolbuffers/protobuf/pull/7288
--- a/cmake/libprotobuf.cmake
```

Let's add the link here to make it at least easier to see, that it was already merged.